src/scripts/types/hg/inventoryItem.ts

```diff
@@ -0,0 +1,14 @@
+
+
+export interface InventoryItem {
+    /** HitGrab's internal number id of item */
+    item_id: number;
+    /** Friendly display name of item */
+    name: string;
+    /** Unique snake_case identifying name of item */
+    type: string;
+    /** Item category: bait, crafting, stat, etc... */
+    // classification: Classification;
+    /** Total amount of item in user inventory */
+    quantity: number;
```

Comment:
sometimes it has a comma and is in quotes - this forces it to turn into a number?

Reply:
This is how we currently view JSON responses from them. We just assume it's a number, but we're going to soon verify that with the Zod library.